Source/WebKit/GPUProcess/graphics/WebGPU/RemoteShaderModule.messages.in

```diff
@@ -24,7 +24,7 @@
 #if ENABLE(GPU_PROCESS)
 
 messages -> RemoteShaderModule NotRefCounted Stream {
-    void CompilationInfo() -> (Vector<WebKit::WebGPU::CompilationMessage> messages) Synchronous
+    void CompilationInfo() -> (Vector<WebKit::WebGPU::CompilationMessage> messages)
```

Comment:
Woah, can we just do this?

Reply:
Yes :), sendWithAsyncReply will complain if the message is marked Synchronous